guides/release/routing/query-params.md

```diff
@@ -51,11 +51,12 @@ array that the `articles` template will render:
 import Controller from '@ember/controller';
 import { computed } from '@ember/object';
 
-export default Controller.extend({
-  queryParams: ['category'],
-  category: null,
+export default class ArticlesController extends Controller {
+  queryParams = ['category'];
+  category = null;
 
-  filteredArticles: computed('category', 'model', function() {
+  @computed('category', 'model')
```

Comment:
Should these `@computed` examples transition to `@tracked`?

Reply:
yes, done